components/profile-controller/pkg/apis/istiorbac/v1alpha1/servicerole_types.go

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
The README could have a short overview on ServiceRole, ServiceRoleBinding, Role, RoleBinding and interactions with the user.

Reply:
updated.